viite-UI/src/view/ProjectEditForm.js

```diff
@@ -371,21 +371,14 @@
       };
 
       var cancelChanges = function() {
-        selectedProjectLinkProperty.setDirty(false);
-        if(projectCollection.isDirty()) {
-          projectCollection.revertLinkStatus();
-          projectCollection.setDirty([]);
-          projectCollection.setTmpDirty([]);
-          projectLinkLayer.clearHighlights();
-          $('.wrapper').remove();
-          formCommon.toggleAdditionalControls();
```

Comment:
So this is not needed any more? Or did this drop out by mistake?

Reply:
I dropped it because it does toggleAdditionalControls() in the reOpenCurrent function which is called on line 381. Viite still works fine, now it just calls toggleAdditionalControls() one time less.